src/coreclr/nativeaot/Runtime/CMakeLists.txt

```diff
@@ -222,6 +220,18 @@ endif()
 
 if(FEATURE_EVENT_TRACE)
   add_definitions(-DFEATURE_EVENT_TRACE)
+  list(APPEND COMMON_RUNTIME_SOURCES
```

Comment:
I assume that Etw events would still be on by default, as opposed to opt in via `EventSourceSupport `flag.

Reply:
Yes, they remain on by default with this change.